aws/rapids_sagemaker_hpo.ipynb

```diff
@@ -50,7 +50,7 @@
   },
```

Comment:
<p>Do we have all these packages/scripts in the unified container? If so, we'd like to skip this step. We can pull/push to ECR for now, and after moving to NGC would ideally eliminate this as well and directly pull from AWS Marketplace</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>